src/main/java/com/ecore/roles/client/model/User.java

```diff
@@ -1,38 +1,27 @@
 package com.ecore.roles.client.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 
 import java.util.UUID;
 
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
 @Builder
-@EqualsAndHashCode
```

Comment:
We can replace the annotations @Getter, @Setter, @EqualsAndHashCode with just @Data